src/PAModelTests/PAModelTests.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
```

Comment:
There is a chance that when the PR action runs, it may still go down the dependency chain for the `Microsoft.NET.Test.Sdk` version `16.5.0` and throw a warning for this issue. Let's go ahead with this change (possibly try consuming into one of the products on a branch, where the test is triggered) and keep an eye out for this warning.

Reply:
Possible, good point.